src/main.rs

```diff
@@ -1,5 +1,9 @@
 use std::io;
 
+fn count_overlap(secret: &str, guess: &str) -> usize {
+    secret.chars().filter(|letter| guess.contains(*letter)).count()
```

Comment:
well I just tried again and now the results are kinda flip flopped, so I dunno... I think the difference is small enough that going with the more "nice" version is worth doing. WDYT?